tools/osbuilder/rootfs-builder/ubuntu/Dockerfile.in

```diff
@@ -33,7 +33,7 @@ RUN apt-get update && \
     $(gcc_arch="@ARCH@" && [ "$(uname -m)" != "$gcc_arch" ] && ( \
          libc_arch="$gcc_arch" && \
          [ "$gcc_arch" = aarch64 ] && libc_arch=arm64; \
-         [ "$gcc_arch" = ppc64le ] && gcc_arch=powerpc64le && libc_arch=ppc64el; \
+         [ "$gcc_arch" = ppc64le ] && gcc_arch=powerpc64 && libc_arch=ppc64el; \
```

Comment:
It seems that this should remain `powerpc64le` since there is `gcc-powerpc64le-linux-gnu` package according to https://www.ubuntuupdates.org/package/core/oracular/main/base/gcc-powerpc64le-linux-gnu, but nothing `powerpc64` (ubuntu and debian dropped support for big endian PowerPC some time back).